complete/_rg

```diff
@@ -170,7 +170,8 @@ _rg() {
     {-w,--word-regexp}'[only show matches surrounded by word boundaries]'
     {-x,--line-regexp}'[only show matches surrounded by line boundaries]'
 
-    + '(zip)' # Compressed-file options
+    + '(input-decoding)' # Compressed-file options
```

Comment:
Missed the comment here

Reply:
Oops.  Did you want that ``-E,--encoding`` option down in that input-decoding group, too?  Instead of "misc/other"?

Reply:
No, these options were grouped together because they're completed exclusively of each other (that's what the `(...)` in the group name means). `-E` is independent and doesn't have any particular relationship with other options, so it can be dumped in with the miscellaneous stuff. I can see how the group name `input-decoding` might kind of imply that relationship, which is why i'd suggested `preprocessor-zip` or something more specific like that; doesn't matter too much tho

Reply:
Ah.  Ok.  I see.  Well, if someone optimizes some other common case and adds another option like --search-pdf (just as an example) we have a general group name.  Doesn't matter to me unless someone complains.